content/02.introduction.md

```diff
@@ -140,6 +140,8 @@ The entry of a coronavirus into a target cell is mediated by the binding of the
 Angiotensin-Converting Enzyme 2 (ACE2) and Transmembrane Serine Protease 2 (TMPRSS2) have been identified as a prime receptor and a critical protease, respectively, facilitating SARS-CoV/CoV-2 entry into a target cell [@doi:10.1038/nature02145; @doi:10.1126/science.abb2507; @doi:10.1128/JVI.01542-10; @doi:10.1016/j.cell.2020.02.052; @doi:10.1128/JVI.02232-10]. 
 This finding has led to a hypothesized role for ACE2 and TMPRSS2 expression in determining which cells, tissues and organs are most likely to be infected by SARS-CoV-2.
 
+However, the use of angiotensin-converting enzyme inhibitors (ACEIs)/angiotensin receptor blockers (ARBs) for hypertension does not appear to be is dangerous in the context of COVID-19, despite the importance of the ACE2 receptor in COVID-19 infection, so patients using these medications have been advised to continue using them [@doi:10.1093/cvr/cvaa097][@doi:10.1001/jama.2020.11301][@doi:10.1001/jama.2020.4812][@doi:10.1161/HYPERTENSIONAHA.120.15082].
```

Comment:
```suggestion
However, the use of angiotensin-converting enzyme inhibitors (ACEIs) and angiotensin receptor blockers (ARBs) for hypertension does not appear to confer additional risk of COVID-19, despite the importance of the ACE2 receptor in COVID-19 infection, and patients using these medications have been advised to continue using them [@doi:10.1093/cvr/cvaa097; @doi:10.1001/jama.2020.11301; @doi:10.1001/jama.2020.4812; @doi:10.1161/HYPERTENSIONAHA.120.15082].
```

Reply:
@dziakj1 does this change sound OK to you? I'm happy to merge in. We could also ask a med student contributor (@scapone01 again?) what they think!

Reply:
Sorry I got a bit busy yesterday. 

Sounds good to me 👍🏼

Reply:
Thanks @rando2 !  But let me look into this if you don't mind.  I thought that the papers were talking about risk of serious vs. mild COVID-19, rather than the risk of infection, because we really don't have data on the latter.

Reply:
@dziakj1 ah I misunderstood your text then! I was just trying to resolve the typo ("does not appear to be is dangerous in the context of COVID-19")

Reply:
Right, we don't have the correct data to answer the question about risk of acquiring COVID-19, which would require a comparison of people exposed and unexposed to ACEis or ARBs and their subsequent risk of acquiring illness.

Instead, we have information from people who are already ill about their exposure status, and the severity of their illness. The exposure is a time-varying factor that may be mis-coded since many people don't take medications as prescribed, and we don't truly know whether or not these drugs impact outcomes. Our clinical trial REPLACE COVID is ongoing, and a clinical trial in Brazil has completed recruitment.

Reply:
More information in our blog on this topic: http://www.nephjc.com/news/covidace2

and in our papers on this topic: @pmid: 32496917 & @pmid: 32220930

The observational studies have quite profound limitations and have suggested a variety of possible answers to whether there is a harm or benefit to ACEis and ARBs. I would say we must await trial results to truly have useful information in hand.

Reply:
I think I wrote a section about this before which probably got removed; it may be possible to look back and see. I'm not sure anything truly substantial has changed, except the earthquake that was Surgisphere's retraction of the NEJM paper on this topic.

Reply:
cc @rando2

Reply:
@byrdjb Uh-oh, I don't think anything got removed on purpose! I'll do a deep dive to figure out what happened. I do remember you writing about this. I have a meeting in a few minutes but I'll ping you all when I figure it out.

Reply:
Thanks, @rando2!

Reply:
@byrdjb we had a bit issue with conflicts in #308 and it seems like some of your text got lost. I am going to go through this very carefully tomorrow but it seems like #290 and #293 got deleted but the rest of the text should be there. I'll add it back in. Thank you SO much for catching this error!!! @dziakj1 #293 addresses some of the same drugs you are talking about here, if you want to take a look!

Reply:
Understood--I am _confident_ that my own unfamiliarity with github contributed, if not caused the whole issue. I had intuited that it had been out of scope for the paper, and I'm glad that may not be the case, since the issue has been given so much attention.

I do believe I wrote that section before the Surgisphere imploded (I reviewed that NEJM paper, raising some doubts, for this project actually!).

Reply:
I am sorry for not including your material, @byrdjb -- I think that it is relevant for this paper, even if it isn't relevant for the Therapeutics section.

Reply:
No worries. If there's something I should do, let me know @rando2, and I will try to do it.

Thanks!

Reply:
@byrdjb I actually think you deserve zero blame for any of the technical issues! The fact that you compartmentalized each topic into discrete PRs made it much easier to diagnose what had gone wrong on my end. Thank you so much for your work on this!